modules/akka-http/README.md

```diff
@@ -0,0 +1,37 @@
+
+# Akka-Http module for PureConfig
```

Comment:
Can you please use tut for the documentation? You should move this to `modules/akka-http/src/main/tut/README.md`, use tags as in [this example](https://github.com/pureconfig/pureconfig/blob/master/modules/akka/src/main/tut/README.md) and run `sbt tut`.

Reply:
sure